modules/drivers/snowflake/resources/metabase-plugin.yaml

```diff
@@ -8,11 +8,20 @@ driver:
   lazy-load: true
   parent: sql-jdbc
   connection-properties:
+    - name: use-account-name
+      type: section
+      default: true
```

Comment:
It would be more natural for me if we flipped the condition, i.e., having `use-host-name` with `default: false`. That would tell me that now one can opt-in to using the host name.